lib/user.rb

```diff
@@ -0,0 +1,27 @@
+require "./lib/menu"
```

Comment:
Nice clean code and some SRP to boot - Its really clear to me what each part is doing!